### ISSUE TITLE
Overwritten anchor names when using "for-each-anchor"

### ISSUE THREAD

**Issue by jrde-88758** [johannes-wolf/cetz#478]
I tried the example code of chapter 3.4.8 and got the same output like shown in the Manual. The names of the north-west and the south-east anchors are overwritten by "end" and "mid". I guess this is not intended(?).

![Overwritten_anchor_names](https://github.com/johannes-wolf/cetz/assets/150175120/7a5d53e1-249b-412b-b196-9c95711d76af)

Hope I could help you, best regards

JR

P.S: I am using the latest compiler version of the "typst.app" and CeTZ-version 0.2.0.

**Comment by fenjalien** [comment]
Yes this is intended, the "end" and "mid" come from path anchors which are new to 0.2. This example could be updated to remove them to demonstrate the `exclude` argument...

**Comment by jrde-88758** [comment]
Thank you for the answer! I searched the manual for the term `exclude` but didn't get any results. Could you tell me please the page where I can find an example of the `exclude`-Argument? For now I used an if conditional to get the output I hoped for.

```
#cetz.canvas({
import cetz.draw: *
  
  grid((-3,-3), (3,3), step: 1cm, stroke: (paint: gray, thickness: 0.4pt))
  
  rect((-2,-2), (2,2), fill: none, name: "E" )
  
  for-each-anchor("E", (name) => {
    if (name != "end" and name != "start" and name != "mid")  {
      content((), box(fill: white, [#name]), angle: -30deg)
    }    
  })
})
```

**Comment by johannes-wolf** [comment]
It is new and not yet in a released version (will come with 0.2.1).

**Comment by jrde-88758** [comment]
Ah, thank you very much for the clarification! Then I will wait for the next release. I wish you a nice evening, JR
